app/views/core/SubVueComponentView.js

```diff
@@ -0,0 +1,55 @@
+import CocoView from 'views/core/CocoView'
+import store from 'core/store'
+
+const silentStore = { commit: _.noop, dispatch: _.noop }
+
+export default class VueComponentView extends CocoView {
+  constructor (options) {
+    super(options)
+
+    if (!this.VueComponent) {
+      throw new Error('Class must be initialized with VueComponent')
+    }
+
+    this.props = {}
+  }
+
+  getMountPoint() {
+    return this.$el[0]
+  }
+
+  buildVueComponent() {
+    return new this.VueComponent({
+      el: this.getMountPoint(),
+      propsData: this.props,
+      store: store
```

Comment:
Would we want to optionally be able to register a functional store? Could this be passed in via the constructor?